test_schedule.njk

```diff
@@ -0,0 +1,173 @@
+---
```

Comment:
The card heights are inconsistent
<img width="2080" height="420" alt="image" src="https://github.com/user-attachments/assets/fbe37717-b787-4d55-8943-2584c81b0c19" />
<img width="2030" height="740" alt="image" src="https://github.com/user-attachments/assets/4d8f5163-249f-47c1-9f5f-de4e9788f901" />
<img width="2028" height="438" alt="image" src="https://github.com/user-attachments/assets/569e98d3-f5e4-4667-8d4b-2c2bf14f101c" />